### ISSUE TITLE
Issue with voice Callflow and Initiate Call

### ISSUE THREAD

**Issue by raja** [messagebird/go-rest-api#28]
We are trying to use the recently added voice package to the call flow api but having issues:

Here is our test function:

```
func (mb *MessageBird) Call(source, destination string) error {
	refCreatedAt, err := time.Parse(time.RFC3339, "2018-01-29T13:46:06Z")
	callflow := voice.CallFlow{
		Title: "Say test",
		Steps: []voice.CallFlowStep{
			&voice.CallFlowSayStep{
				Voice:    "female",
				Payload:  "This is a journey into the sound of voice!",
				Language: "en-US",
			},
			&voice.CallFlowPauseStep{
				Length: time.Second * 4,
			},
			&voice.CallFlowHangupStep{},
		},
		CreatedAt: refCreatedAt,
		UpdatedAt: refCreatedAt,
	}

	call, err := voice.InitiateCall(mb.client, source, destination, callflow, nil)
	log.Printf("%+v", call)
	return err
}
```

And the error:

```
[18:32:31][THEHIRINGCO] : 2018/04/17 18:32:31 unable to parse CallFlow CreatedAt: parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"
[18:32:31][THEHIRINGCO] : panic: unable to parse CallFlow CreatedAt: parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"
```

**Comment by raja** [comment]
We have no issues with sending/receiving sms.

**Comment by raja** [comment]
I also believe the text in the errors around line https://github.com/messagebird/go-rest-api/blob/5a6ec99e83b44ad3e9bcd15c400d6b500032247d/voice/call.go#L79 should have 'Call' instead of Callflow to differentiate from callflow.go

**Comment by polyfloyd** [comment]
Hi,

Thanks for pointing this out! The issue has been resolved in #29.
